src/styles/icon/_icons.scss

```diff
@@ -0,0 +1,8 @@
+.nhsapp-icon {
```

Comment:
```sass
.nhsapp-icon {
 // styles
 &__unread-indicator {
  // styles
 }
}
```
The `&` can make this a little cleaner meaning it's easier to add/extend in the future.